java-test-samples/step-functions-local/README.md

```diff
@@ -0,0 +1,28 @@
+# Step Functions Local Testing using JUnit and Spock
+This project contains information about the [sample test application](https://github.com/aws-samples/aws-stepfunctions-examples/tree/main/sam/demo-local-testing-using-java) that highlights use of JUnit and Spock test framework to test Step functions locally.
```

Comment:
Capitalize F in Step Functions

Reply:
Worth adding this docs link? https://docs.aws.amazon.com/step-functions/latest/dg/sfn-local.html

Reply:
Updated